spec/Classes.tex

```diff
@@ -359,7 +422,79 @@ \subsection{Class Methods}
 \index{secondary methods}
 
 Methods on classes are referred to as to as \emph{class methods}.
-See the methods section~\rsec{Methods}.
+See the methods section~\rsec{Methods} for more information about methods.
```

Comment:
Or simply:

`Methods on classes (\rsec{Methods}) are referred to as to as \emph{class methods}.`

?

Reply:
I think I like the longer one better on this one.